app/controllers/keywords_controller.rb

```diff
@@ -25,6 +25,14 @@ def create
     redirect_to keywords_path
   end
 
+  def show
+    keyword = Keyword.includes(:links).find(params[:id])
+    presenter = KeywordPresenter.new(keyword)
```

Comment:
Let's add a newline after this line

Reply:
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)